ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.12.yaml

```diff
@@ -104,7 +104,7 @@ tests:
     test:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
-    workflow: cucushift-installer-rehearse-aws-ipi-localzone
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-localzone
```

Comment:
The test name `aws-ipi-localzone-p2-f7` need to reflect the change, too, as we use that to identify which profile we are testing in ReportPortal.

Reply:
I think that ovn is the default network for 4.12+,  so the test name should be ok without the network type.

Reply:
There are changes from version to version.
Prefer let the name reflect what kind of tests it is. 
It's clear that we know what's changed from 4.11 to 4.12 because we are heavily on 4.12. How about several release later ? E.g, can you recall all the changes we have from 4.7 to 4.8 ?

Reply:
Get it, will update it to reflect workflow configuration.